src/transformation/visitors/expression-statement.ts

```diff
@@ -4,8 +4,14 @@ import { FunctionVisitor } from "../context";
 import { transformBinaryExpressionStatement } from "./binary-expression";
 import { transformLuaTableExpressionStatement } from "./lua-table";
 import { transformUnaryExpressionStatement } from "./unary-expression";
+import { transformMultiDestructuringAssignmentStatement } from "./language-extensions/multi";
 
 export const transformExpressionStatement: FunctionVisitor<ts.ExpressionStatement> = (node, context) => {
+    const multiResult = transformMultiDestructuringAssignmentStatement(context, node);
```

Comment:
Would prefer if you only call this `transformMultiDestructuringAssignment` function after checking `isMultiFunctionNode` (same for function.ts)